docs/marketplace/add_to_marketplace/contributing_overview.md

```diff
@@ -1,61 +1,52 @@
 ---
 id: contributing_overview
-title:  Contributing to the Marketplace
-sidebar_label:  Contributing to the Marketplace
+title:  Manage your Company Marketplace
+sidebar_label:  Internal Company Marketplace
 ---
 
-As a Console user, you can contribute to the Marketplace by creating new items of any kind. You can choose to make them available on the internal Company, or request for them to be published.
-
-In this section, we dive into the process of creating a new item for the Marketplace, analyzing the main differences between different types of items and providing some configuration examples.
-
-:::info
-
-There are different methods to manage Marketplace items, described in details in the [related page](/marketplace/add_to_marketplace/manage_marketplace_items.md)
-
-:::
+In this section, we dive into the process of creating a new item for your **Internal Company Marketplace**, analyzing the main differences between different types of items and providing some configuration examples.
 
 ## Public and Internal Company Marketplace
 
+As a Console user, you can access both Mia-Platform public Marketplace catalog and your Internal Company Marketplace. 
+While the Mia-Platform public Marketplace is composed by all the items publicly available in the Mia-platform catalog, the Internal Company Marketplace is a Company-specific private space where to create and manage the items which are available only for your Company.
```

Comment:
I would move this below the public marketplace explanation, and use bold for the name
```suggestion
While the Mia-Platform public Marketplace is composed by all the items publicly available in the Mia-platform catalog, the **Internal Company Marketplace** is a Company-specific private space where to create and manage the items which are available only for your Company.
```

Reply:
As mentioned in the title, the goal is to make the user focus on the concept of the Marketplace of the Company